src/server/common/helpers/errors.test.js

```diff
@@ -107,14 +104,40 @@ describe('#catchAll', () => {
     // @ts-expect-error - Testing purposes only
     catchAll(mockRequest(statusCodes.imATeapot), mockToolkit)
 
-    expect(mockErrorLogger).toHaveBeenCalledWith(mockStack)
     expect(mockToolkitView).toHaveBeenCalledWith(errorPage, {
       pageTitle: 'Something went wrong',
       heading: statusCodes.imATeapot,
       message: 'Something went wrong'
     })
     expect(mockToolkitCode).toHaveBeenCalledWith(statusCodes.imATeapot)
   })
+
+  const statuses4xx = [
+    statusCodes.notFound,
+    statusCodes.forbidden,
+    statusCodes.unauthorized,
+    statusCodes.imATeapot
+  ].map((status) => [status])
+
+  test.each(statuses4xx)('should call warn logger on 4xx', (status) => {
```

Comment:
should we use "it" here? same on line 134

Reply:
agreed - will convert the whole file over.